src/devices/Nmea0183/Sentences/HeadingTrue.cs

```diff
@@ -0,0 +1,109 @@
+    /// not equal to the heading.
+    /// </summary>
+    public class HeadingTrue : NmeaSentence
+    {
+        /// <summary>
+        /// This sentence's id
+        /// </summary>
+        public static SentenceId Id => new SentenceId("HDT");
+        private static bool Matches(SentenceId sentence) => Id == sentence;
+        private static bool Matches(TalkerSentence sentence) => Matches(sentence.Id);
+
+        /// <summary>
+        /// Constructs a new MWV sentence
+        /// </summary>
+        public HeadingTrue(double angle)
+            : base(OwnTalkerId, Id, DateTimeOffset.UtcNow)
+        {
+            Angle = Angle.FromDegrees(angle);
+            Valid = true;
+        }
+
+        /// <summary>
+        /// Internal constructor
+        /// </summary>
+        public HeadingTrue(TalkerSentence sentence, DateTimeOffset time)
+            : this(sentence.TalkerId, Matches(sentence) ? sentence.Fields : throw new ArgumentException($"SentenceId does not match expected id '{Id}'"), time)
+        {
+        }
+
+        /// <summary>
+        /// Date and time message (ZDA). This should not normally need the last time as argument, because it defines it.
+        /// </summary>
+        public HeadingTrue(TalkerId talkerId, IEnumerable<string> fields, DateTimeOffset time)
+            : base(talkerId, Id, time)
+        {
+            IEnumerator<string> field = fields.GetEnumerator();
+
+            double? angle = ReadValue(field);
+            string reference = ReadString(field) ?? string.Empty;
+
+            // The HDT sentence must have a "T" (True) reference, otherwise something is fishy
+            if (reference == "T" && angle.HasValue)
+            {
+                Angle = Angle.FromDegrees(angle.Value);
+                Valid = true;
+            }
+            else
+            {
+                Angle = Angle.Zero;
+                Valid = false;
+            }
+        }
+
+        /// <summary>
+        /// This is true for this message type
+        /// </summary>
+        public override bool ReplacesOlderInstance => true;
+
+        /// <summary>
+        /// Angle to report (0: North, 90 East, etc.)
+        /// </summary>
+        public Angle Angle
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// Presents this message as output
+        /// </summary>
+        public override string ToNmeaParameterList()
+        {
+            if (Valid)
+            {
+                return FormattableString.Invariant($"{Angle.Normalize(true).Degrees:F1},T");
+            }
+
+            return string.Empty;
+        }
+
+        /// <inheritdoc />
+        public override string ToReadableContent()
+        {
+            if (Valid)
+            {
+                return $"True Heading: {Angle.Normalize(true).Degrees:F1}°";
```

Comment:
same here for the degree, should be unicode encoded. I guess it should appear elsewhere as well